src/components/Navbar/Dropdown.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Dropdown as AntdDropdown } from "antd";
+
+const { white } = uiColors;
+
+interface DropdownProps {
+  dataCy?: string;
+  menuItems: JSX.Element;
+  title: string;
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  dataCy,
+  menuItems,
+  title,
+}) => (
+  <AntdDropdown overlay={menuItems}>
```

Comment:
This might be out of scope for this ticket if we are already using this in other parts of the header, but I know that we are trying to move away from using antd components.

Reply:
Yeah could you replace this with a leafygreen menu https://www.mongodb.design/component/menu/example/

Reply:
Good idea, I'll take a stab at this!